src/sentry/static/sentry/app/components/assistant/getGuidesContent.tsx

```diff
@@ -210,3 +214,102 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
     },
   ];
 }
+
+function getDemoModeGuides(): GuidesContent {
+  return [
+    {
+      guide: 'sidebar',
+      requiredTargets: ['projects', 'issues'],
+      priority: 2,
+      steps: [
+        {
+          title: t('Projects'),
+          target: 'projects',
+          description: t(
+            `Projects allow you to scope events to a distinct application in your organization.`
+          ),
+        },
+        {
+          title: t('Issues'),
+          target: 'issues',
+          description: t(
+            `A collection of error events reported as your customers experience them.`
+          ),
+        },
+        {
+          title: t('Performance'),
+          target: 'performance',
+          description: t(
+            `Sentry tracks your software performance, measuring metrics like throughput and latency, and displaying the impact of errors across multiple systems.`
+          ),
+        },
+        {
+          title: t('Releases'),
+          target: 'releases',
+          description: t(
+            `Track the health of your releases down to the moment it starts to erode.`
+          ),
+        },
+        {
+          title: t('Discover'),
+          target: 'discover',
+          description: t(
+            `Query and unlock insights into the health of your entire system and get answers to critical business questions -- all in one place.`
+          ),
+        },
+      ],
+    },
+    {
+      guide: 'issue_stream_v2',
+      requiredTargets: ['issue_title'],
```

Comment:
`issue_title` might not be the right target for issue list, since the component is also in issue details. `issue_stream` could be used instead.

Reply:
@maheskett Yea, true. I could pass a prop for the guide anchor name so they are different.

Reply:
Added a prop here